### PR TITLE
Remove viewpoints atom type

### PR DESCRIPTION
**Pull request by tomrf1**
I'll put this in the next major release, along with the thriftification and media atom changes

### CODE REVIEW

File: models/src/main/thrift/content/v1.thrift [hunk 1]
```diff
@@ -1119,9 +1119,7 @@ struct Section {
 struct Atoms {
     1: optional list<contentatom.Atom> quizzes
 
-    2: optional list<contentatom.Atom> viewpoints
-
-    3: optional list<contentatom.Atom> media
+    2: optional list<contentatom.Atom> media
```
Comment (cb372) [inline models/src/main/thrift/content/v1.thrift]: Let's keep `media` as field number 3 and leave the viewpoints line commented out as a reminder that we should never reuse field number 2. Reusing field numbers is a big no-no in Thrift.

File: models/src/main/thrift/content/v1.thrift [hunk 2]
```diff
@@ -1401,9 +1399,7 @@ struct ItemResponse {
     /* New story packages */
     20: optional list<Package> packages
 
-    21: optional list<contentatom.Atom> viewpoints
-
-    22: optional contentatom.Atom media
+    21: optional contentatom.Atom media
```
Comment (cb372) [inline models/src/main/thrift/content/v1.thrift]: Same as above

**Comment by cb372** [comment]
👍  (although I would have left a comment in the Thrift file documenting what the missing field was)

**Comment by tomrf1** [comment]
We decided in the end to leave the viewpoints fields in the thrift model

**Pull request merged**